docs/Arguments.md

```diff
@@ -131,6 +131,10 @@ usage: -m llumnix.entrypoints.vllm.api_server [-h]
 - Possible choices: gloo, rpc
 - Default: "rpc"
 
+`--migration-backend-init-timeout`
+- Timeout(s) for initializing migration backend.
+- Default: 5.0
```

Comment:
5s seems too short to me. It will be insufficient when the cluster is large. I'd recommend at least 10s or more.

Reply:
get it, set it to 10s.